app.json

```diff
@@ -5,6 +5,12 @@
     "CHECK_BROKEN_LINKS": "no",
     "YARN_PRODUCTION":{
       "required": true
+    },
+    "HEROKU_APP_NAME": {
+      "required": true
+    },
+    "HEROKU_PARENT_APP_NAME": {
+      "required": true
     }
```

Comment:
What does `HEROKU_PARENT_APP_NAME` do?

Reply:
Ah yeah, that was from an earlier approach. I don't actually need it!

Reply:
To answer this more directly:

We're (ab)using a feature of Heroku called "review applications" which uses a base application (that we don't actually deploy), and then spins up a special type of application for each PR opened (these are the review applications). The parent app in this context is the name of the app that this is "previewing".